src/form/DateInputFormField.tsx

```diff
@@ -54,6 +54,7 @@ const DateInputFormField = ({
       language={i18n.language as Language}
       invalid={meta.touched && meta.invalid}
       errorText={meta.touched && meta.error}
+      crossOrigin={undefined}
```

Comment:
It doesn't make sense to me why and at what point in the type dependency chain this becomes a required field, because this shouldn't be needed to specify. But this works as a fix, so no problem.

Reply:
yup. This was a weird one